_newsletters/2023-03-14-february-2023.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+category: newsletters
+title: February 2023 Newsletter
+filename-custom: february-2023
+last_modified_at: March 14, 2023
+sidenav: about_us
+newsletter-year: 2023
+description: ""
+---
+## Faceted search for API customers
+
+### Now available!
+
+Search filtering (faceted search) is now available via our Results API! You will be able to offer filters on your results page using text fields from our index, such as tags, audience, content type, and custom fields. [Read our API](https://open.gsa.gov/api/searchgov-results/) documentation and reach out to our team [search@gsa.gov](search@gsa.gov) if you would like to start setting this feature up on your website.
+[Read our API documentation](https://open.gsa.gov/api/searchgov-results/)
```

Comment:
Please add a blank line before this line, that will create the paragraph break on the page.